mobile/src/bundles/talent/screens/talent-profile/talent-profile.tsx

```diff
@@ -27,14 +35,29 @@ const TalentProfile: React.FC = () => {
                     globalStyles.pv25,
                     globalStyles.pl25,
                     globalStyles.pr10,
-                    styles.header,
                     globalStyles.flexDirectionRow,
                     globalStyles.justifyContentSpaceBetween,
                     globalStyles.alignItemsCenter,
+                    styles.header,
                 ]}
             >
                 <Text category={TextCategory.H3}>Your profile</Text>
-                {!isApproved && <VerificationMessage />}
+                <View
+                    style={[
+                        globalStyles.flexDirectionRow,
+                        globalStyles.justifyContentFlexEnd,
+                        globalStyles.alignItemsCenter,
+                    ]}
+                >
+                    {!isApproved && <VerificationMessage />}
+                    <Pressable onPress={handleLogout}>
+                        <CommunityIcon
+                            name={IconName.LOGOUT}
+                            size={30}
+                            color={Color.TEXT2}
+                        />
+                    </Pressable>
```

Comment:
We can move the logout button to a shared component and also keep the dispatch of that action, but if you have some higher priority tasks for now, let's leave it as it is

Reply:
I'll do it in another branch, but later